iota/transaction/base.py

```diff
@@ -69,16 +73,20 @@ def __init__(
       signature_message_fragment,
       address,
       value,
-      tag,
+      legacy_tag,
```

Comment:
In a future snapshot (probably the next one), `legacy_tag` will be removed entirely and replaced with the new `tag`.

For now, let's just treat `legacy_tag` as an alias to new `tag`:

- Initialiser accepts `tag` only (no `legacy_tag`).
- Set `self.tag` as normal.
- Make `self.legacy_tag` an alias for `self.tag` (recommend using a `@property` for that, just in case).

Reply:
I left legacy_tag in the initializer defaulted to None. The reason being when creating the transaction from the tryte string legacy_tag is extracted and needs a way to be set. It gets set that way in the JS library so I figured it would be best to keep it consistent. If legacy_tag is not set it will just point to the tag.